### DISCUSSION TITLE
React Characters Remaining - Jan 10

### DISCUSSION THREAD

**Discussion by ankushdharkar** [RealDevSquad/Centauri-React-Native-Cohort#8] [Show and tell]
Were you able to get it working? Post a screen recording

**Comment by tenkunkyab** [comment]
Haven't done a lot on the styling bit, yet. 

https://user-images.githubusercontent.com/5739849/148493742-b697cc92-f695-4fde-8e49-965a5430a820.mp4

**Comment by puneetsingh99** [comment]
https://user-images.githubusercontent.com/75030119/148500294-370bd81d-54bd-434d-a3e4-3a773a1218b4.mov

**Comment by lyqht** [comment]
> Videos below are recorded on my android mobile phone 😆 pardon the timer button at the side.

- Here, I have set the max limit of characters to 10 for simplicity of demo

https://user-images.githubusercontent.com/35736525/148541529-4f45be4e-b4c9-4f4c-90f5-feded1ed5411.mp4

---

- When I tried it with max limit of 280 characters, it still works as intended. 
- I find it really interesting how the TextInput component still keeps the bold formatting that I copied elsewhere 😃

https://user-images.githubusercontent.com/35736525/148542454-e93cc50b-ade2-449b-883b-7c17cd0db115.mp4

The implemented component be found [here](https://github.com/lyqht/centauri-react-native/blob/main/UserGreeter/components/Textbox.tsx).

**Comment by vp5h** [comment]
- here's the [Snack](https://snack.expo.dev/@vp5h/tweet-rn)
![rn1](https://user-images.githubusercontent.com/54497006/148544203-85d5d666-4044-4f6a-b0b8-4950bc21ce19.gif)

**Comment by priyanshk20** [comment]
![tweetcount](https://user-images.githubusercontent.com/52736997/148548369-6fc20742-65c7-4c19-a9c4-4b98f16057fc.gif)

**Comment by ghost** [comment]
https://user-images.githubusercontent.com/96296872/148552193-c8d6c552-41f5-42e3-abf9-ba260f053e32.mp4

**Comment by pallabez** [comment]
https://user-images.githubusercontent.com/63191499/148553057-cfce470a-c269-4099-9ead-c3a0d95f7dff.mp4

**Comment by SaiKarthikYamavaram** [comment]
https://user-images.githubusercontent.com/37267369/148575679-d275b401-7c5b-42ba-977c-fa2c31a57c30.mp4

**Comment by tewarig** [comment]
https://user-images.githubusercontent.com/54894783/148584505-86fcc404-8352-4966-9c70-f7e9e82e6eda.mp4

**Comment by manmodesanket** [comment]
https://user-images.githubusercontent.com/30385252/148639814-c4604216-ac96-4838-99a7-2673483b96f4.mov

Here is the repo: https://github.com/manmodesanket/react-native-characters-left

**Comment by pruthvirajmv** [comment]
Here is my attempt for characters remaining task and the code is [here](https://github.com/pruthvirajmv/react-native-practice). 

![RDS_character_remaining](https://user-images.githubusercontent.com/74802690/148642297-28a00b52-e609-4d38-bce4-81a92dfbab67.gif)

**Comment by akslucky619** [comment]
https://user-images.githubusercontent.com/48199192/148643770-bff3c51c-bab2-405e-b000-1646d42b864c.mp4

Here's my attempt.

**Comment by rohan9896** [comment]
https://user-images.githubusercontent.com/56235204/148645779-f9789d80-b3e1-4330-aa1f-e59b6f87c3b0.mov

here is the repo - https://github.com/rohan9896/Centauri-react-native

**Comment by Joyan11** [comment]
https://user-images.githubusercontent.com/23113259/148647842-d69dc15d-f026-4662-b5fa-b1abbefd2302.mov

**Comment by jatin33** [comment]
https://user-images.githubusercontent.com/13780467/148647694-9650a30a-1454-4ba6-b2d3-880fca281fc3.mov

**Comment by nikmish1** [comment]
https://drive.google.com/file/d/1DebgmM0SqXcTuWbzxdhihXUIumIqX2eq/view?usp=sharing

**Comment by nandanpandey43** [comment]
Tweet counter App Demo video
Source Code :-https://github.com/nandanpandey43/RDS-React-Native-Cohort-Apps/tree/master/TweetCounterApp

https://user-images.githubusercontent.com/38531527/148671582-66a58805-1aea-4699-b7ba-89cf97163969.mp4

.

**Comment by Pratim-Kumar-Hazarika** [comment]
Character Count App
Repo Link : https://github.com/Pratim-Kumar-Hazarika/CentauriReactNative/tree/character-count

https://user-images.githubusercontent.com/75177895/148672956-db7d1d00-8f13-413a-bceb-17e77b5bfee4.mov

**Comment by jitendraag** [comment]
App:
Repo link: https://github.com/jitendraag/CharactersRemainingCentauri
https://user-images.githubusercontent.com/3295926/148674893-23f38223-83f8-4266-8994-2bfaf6990e03.mov

**Comment by lakshayman** [comment]
TweetApp (Character Remaining App):
Repo Link: https://github.com/lakshayman/CentauriTweetApp

https://user-images.githubusercontent.com/45519620/148676250-b8b4d21a-c5b4-4460-94a6-9c6178d6196b.mp4

**Comment by harishfalco** [comment]
Donw with the assignment
repo link - [ https://github.com/harishfalco/character-counter-app
](url)
![image](https://user-images.githubusercontent.com/62054469/148677958-b33306fb-6f7a-47b1-8d54-a0cef53059d2.png)

**Comment by abhay05** [comment]
Done with the assignment
repo link -> https://github.com/abhay05/

https://user-images.githubusercontent.com/23046900/148680462-86856f28-e022-469c-abef-4237b33e7217.mp4

Centauri-React-Native-Cohort/tree/feature/twitter_tweet_box

**Comment by Sahil-Bhatia** [comment]
Done with tweet character count assignment.
Repo: https://github.com/Sahil-Bhatia/tweet-character-count.git

https://user-images.githubusercontent.com/31650286/148682642-e166b7f7-5c21-444a-a554-1fbab12f3b92.mp4

**Comment by vinayak-trivedi** [comment]
https://user-images.githubusercontent.com/90315175/148682920-0a7c7aef-9611-4cd8-b13e-957715670088.mp4

**Comment by dineshkumar181094** [comment]
https://github.com/dineshkumar181094/CentauriReactNative-Char-Counter






https://user-images.githubusercontent.com/36758667/148685090-69f58947-0d31-44da-a2da-dcf3ab482561.mp4

**Comment by mohitshende** [comment]
https://user-images.githubusercontent.com/54403334/148686405-be13fc8a-00a4-4d1d-9821-648c0f7561c6.mp4

**Comment by mohankumar80** [comment]
https://user-images.githubusercontent.com/50794740/148689893-55e16356-6b6f-4ba2-8553-d124da254aed.mp4

Done with twitter characters count assignment.

**Reply by shreya-mishra** [reply]
Done with the Tweet_Value_Count

https://www.loom.com/share/b7e38e1499684391b44c259cc99cd439

**Comment by NaelaTarannum** [comment]
[

https://user-images.githubusercontent.com/24935748/148725329-4c839894-10e1-4b11-8dee-f41ad5b3c0b6.mov

](url)

**Comment by chaitanyaj2222** [comment]
https://user-images.githubusercontent.com/56331036/148727295-87c88ca0-7b19-4c24-8f41-7a50bb6de17c.mp4

**Comment by prachic17** [comment]
Here is my tweet count assignment

https://user-images.githubusercontent.com/61531836/148757223-1e3d4ae1-b665-41bd-a388-3b641ab90735.mov

**Comment by shaksham08** [comment]
Here is My tweet count assignment

https://github.com/shaksham08/RDS-reactnative-tweetcharcounter

https://user-images.githubusercontent.com/33754070/148766484-5c8817e2-0724-4738-8ea8-c09fbb692bec.mp4

**Comment by Arnab-31** [comment]
https://user-images.githubusercontent.com/57807670/148798701-01209469-c001-408d-915a-4f21949055ba.mp4

https://github.com/Arnab-31/Tweet-Character-Counter

**Comment by shubham-ghuge** [comment]
https://user-images.githubusercontent.com/48974345/148799397-252c77dc-46f4-4a34-8609-9d4bff9064be.mov

**Comment by asyriac** [comment]
https://user-images.githubusercontent.com/43615822/148802919-e4e414aa-e365-43c3-b7ee-315293ede37b.mp4

**Comment by Ramanpreet6262** [comment]
Done with Characters Remaining task.
https://github.com/Ramanpreet6262/Centauri-Characters-Remaining


https://user-images.githubusercontent.com/28749866/148823935-042dc0b6-f63b-4c0a-810d-a55b61553a48.mp4

**Comment by vaishnavme** [comment]
![Screenshot_1641874082](https://user-images.githubusercontent.com/42497931/148880511-9995a822-9634-4c61-b8f6-5dc610c905bf.png)

PR: https://github.com/vaishnavme/baseOne/pulls

**Comment by aakankshadhurandhar** [comment]
https://user-images.githubusercontent.com/34390765/148985913-9792da87-896d-4dee-8e8c-37c55d4daac4.mp4

Repo : [https://github.com/aakankshadhurandhar/React-Native-Corhort/tree/Task-2---Count-no-of-letters](https://github.com/aakankshadhurandhar/React-Native-Corhort/tree/Task-2---Count-no-of-letters)

**Comment by dhanushkumar-dev** [comment]
https://user-images.githubusercontent.com/79807326/149358483-67f0078c-08b8-44ce-a5e3-19e95678cc52.mp4

**Comment by ParakhBansal** [comment]
https://user-images.githubusercontent.com/65717231/149621862-b315e519-179c-4161-8bd6-c4038b7492c0.mp4
